tests/unit/amo/components/TestRatingManager.js

```diff
@@ -36,7 +37,8 @@ function render(customProps = {}) {
   const props = {
     AddonReview: () => <div />,
     AuthenticateButton: () => <div />,
-    addon: fakeAddon,
+    ReportAbuseButton: () => <div />,
```

Comment:
I gave myself 30 minutes to try converting this pre-enzyme suite to `shallowUntilTarget` but it was not trivial.

Reply:
Fair enough. 😄